docs/lib/getting-started/fragments/android/add-api.md

```diff
@@ -1,42 +1,205 @@
+
+1. Review the new user details. When finished, press **Create user**.
+
+1. Go back to your terminal and press **Enter**. Next, you will be prompted for credentials for the new user. Copy and paste the credentials for this user when prompted and name your profile **Amplify**.
+
+    ```console
+    Enter the access key of the newly created user:
+    ? accessKeyId:  [hidden] 
+        ********************
+    ? secretAccessKey: [hidden]
+        ****************************************
+    ? Profile Name:  (default)
+        amplify
+
+    Successfully set up the new user.
+    ```
+
+1. In your terminal, ensure you are in the root of your project directory. Run `amplify init`. Enter a name for your environment such as *tutorial*. Select your preferred editor, and your AWS profile:
+
+    ```console
+    ? Enter a name for the environment
+        tutorial
+    ? Choose your default editor:
+        IntelliJ IDEA
+    ? Do you want to use an AWS profile?
+        Yes
+    ? Please choose the profile you want to use
+        amplify
+
+    Adding backend environment dev to AWS Amplify Console app: abcd1234efg567
+    ⠋ Initializing project in the cloud...
+    ```
+
+    This initializes your Amplify project on the backend
+
+1. In the terminal, run `amplify push`. You can select the defaults to the prompts:
+
+    ```console
+    ✔ Successfully pulled backend environment dev from the cloud.
+
+    Current Environment: dev
+
+    | Category | Resource name     | Operation | Provider plugin   |
+    | -------- | ----------------- | --------- | ----------------- |
+    | Api      | amplifyDatasource | Create    | awscloudformation |
+    
+    ? Are you sure you want to continue?
+        Yes
+    ? Do you want to generate code for your newly created GraphQL API
+        Yes
+    ? Enter the file name pattern of graphql queries, mutations and subscriptions
+        app/src/main/graphql/**/*.graphql
+    ? Do you want to generate/update all possible GraphQL operations - queries, mutations and subscriptions
+        Yes
+    ? Enter maximum statement depth [increase from default if your schema is deeply nested]
+        2
+        
+    ⠋ Updating resources in the cloud. This may take a few minutes...
+    ```
+
+1. Run your application. This will create a new Todo and synchronize it to your API.
+
+1. In the terminal, run `amplify api console`. When prompted, select **GraphQL**. This will open the AWS AppSync console.
+
+    ```bash
+    amplify api console
+    ```
+
+    ```console
+    ? Please select from one of the below mentioned services: (Use arrow keys)
+       GraphQL 
+    ```
+
+1. Copy and paste the following query:
+
+    ```graphql
+    query GetTodos {
+      listTodos {
+        items {
+          id
+          name
+          description
+        }
+      }
+    }
+    ```
+
+1. Press the **play button** to run the query. This will return all of the synchronized Todos:
```

Comment:
i'm guessing there might be some steps missing here? like a screenshot? etc?

Reply:
Hmm, no. See the below screenshot. What do you think should be added?

Reply:
I think there should be a screenshot like this:
![ss](https://user-images.githubusercontent.com/1911939/81618568-79734480-939c-11ea-89cb-150c5a1a6152.png)
(and maybe an arrow on the play button)